op-devstack/devtest/testing.go

```diff
@@ -325,13 +312,13 @@ func SerialT(t *testing.T) T {
 		span.End()
 	})
 
-	// Set the lowest default log-level, so the log-filters on top can apply correctly
-	logger := testlog.LoggerWithHandlerMod(t, log.LevelTrace,
+	// Set the default log-level to INFO, so the log-filters on top can apply correctly
+	logger := testlog.LoggerWithHandlerMod(t, log.LevelInfo,
```

Comment:
I think this creates the same kind of problem as we had in the orchestrator package-wide logger, where the log-level of the inner log handler stops the below-info logs from going through at all.

Reply:
Yup, makes sense. Reverting these lines.